analytics/event.go

```diff
@@ -0,0 +1,31 @@
+package analytics
+
+import "time"
+
+// Holds the name and properties of an analytics event.
+type Event struct {
+	// The name of the event.
+	name string
+
+	// The timestamp of the event. Set to nil to use the current time.
+	timestamp time.Time
```

Comment:
Do you mean "set to 0"? `nil` is not a valid value for `time.Time`.

Reply:
Oops, I never removed this comment from an earlier revision. Fixed in [94aedbe](https://github.com/akitasoftware/akita-libs/pull/152/commits/94aedbee95b665ccf0152cdfc4b3bbb4a01a8b46)